app/scripts/controllers/gene.js

```diff
@@ -98,71 +98,277 @@ angular.module('oncokbStaticApp')
                 }
             }
         };
+        $scope.variant = {
+            name: $routeParams.variantName,
+            oncogenic: '',
+            mutationEffect: '',
+            highestLevel: '',
+            mutationSummary: '',
+            displayName: $routeParams.variantName,
+            relevantVariants: [],
+            loadingRelevantAlts: false,
+            generalMutations: ['Oncogenic Mutations', 'Gain-of-function Mutations', 'Loss-of-function Mutations', 'Switch-of-function Mutations', 'Likely Oncogenic Mutations', 'Likely Gain-of-function Mutations', 'Likely Loss-of-function Mutations', 'Likely Switch-of-function Mutations'],
```

Comment:
The `generalMutations` should come from back-end. A new endpoint needs to be created.